crates/containerd-shim-wasm/src/sandbox/containerd.rs

```diff
@@ -0,0 +1,196 @@
+#![cfg(unix)]
+use artifact_spec::image::ImageManifest;
+use containerd_client;
+use containerd_client::services::v1::containers_client::ContainersClient;
+use containerd_client::services::v1::content_client::ContentClient;
+use containerd_client::services::v1::images_client::ImagesClient;
+use containerd_client::services::v1::{
+    GetContainerRequest, GetImageRequest, ReadContentRequest, ReadContentResponse,
+};
+use containerd_client::tonic::transport::Channel;
+use containerd_client::tonic::{self, Streaming};
+use containerd_client::with_namespace;
+use tokio::runtime::Runtime;
+use tonic::Request;
+
+use crate::sandbox::error::{Error as ShimError, Result};
+use crate::sandbox::oci::{self, OciArtifact};
+
+pub struct Client {
+    inner: Channel,
+    rt: Runtime,
+    namespace: String,
+}
+
+// sync wrapper implementation from https://tokio.rs/tokio/topics/bridging
+impl Client {
```

Comment:
The user here is our Sandbox instance which is sync. The idea was since this the only use I wanted to avoid polluting the sandbox instance with the async related code there. But this module has evolved a bit from when I first implemented it and has become simpler.  It might not be much to wrap it at that layer.  WDYT? I don't know that I have an opinion here.

Reply:
Thanks for the explaination. I am okay with keeping it as is